homeassistant/components/camera/arlo.py

```diff
@@ -153,3 +158,21 @@ def disable_motion_detection(self):
         """Disable the motion detection in base station (Disarm)."""
         self._motion_status = False
         self.set_base_station_mode(ARLO_MODE_DISARMED)
+
+    @Throttle(UPDATE_THROTTLE_INTERVAL)
```

Comment:
Overriding `SCAN_INTERVAL` is better, right @pvizeli?

Reply:
Scratch that, it wouldn't handle multiple entity calls.

Reply:
Unfortunately, `Throttle` doesn't appear to be the right solution either: with it in place in my 4-camera setup, only 1 of the cameras ever receives its attribute updates; the others continue to carry `null`. Wondering if `Throttle` is somehow applying to _all_ entities?

Reply:
How do you want to throttle the calls? Should each entity be allowed a call per time unit or do you want one call per time unit for all entities. The latter will require a separate data class that caches the result of the call. In this case use `Throttle`. In the first case, I think, you should OVERRIDE the module constant `SCAN_INTERVAL`. Let's wait for @pvizeli recommendation though.

Reply:
I like the former. Will await @pvizeli’s sage advice. 😎

Reply:
SCAN_INTERVAL is the solution :)